comtypes/client/_generate.py

```diff
@@ -246,9 +249,9 @@ def _create_wrapper_module(tlib, pathname):
     return _create_module_in_file(modulename, code)
 
 
-def _get_known_symbols():
-    # type: () -> dict[str, str]
-    known_symbols = {}  # type: dict[str, str]
+def _get_known_symbols() -> Dict[str, str]:
+
+    known_symbols: Dict[str, str] = {}
```

Comment:
This package still supports Python 2.7.

You should write type annotations by comments in the way of [PEP484](https://peps.python.org/pep-0484/#suggested-syntax-for-python-2-7-and-straddling-code).

For example, here will be...
```py
def _get_known_symbols():
    # type: () -> Dict[str, str]
    known_symbols = {}  # type: Dict[str, str]
    ...
```